new-log-viewer/package.json

```diff
@@ -5,8 +5,6 @@
   "main": "src/index.tsx",
   "scripts": {
     "build": "webpack --config webpack.prod.js",
-    "lint:check": "npx eslint --format github --no-eslintrc --config package.json ./src",
```

Comment:
Changes in this file are experimental and not at all related to the pull request at https://github.com/y-scope/yscope-log-viewer/pull/57